packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -121,7 +121,7 @@ export class ConnectionService {
 
     connectionRequest.setThread({
       threadId: connectionRequest.threadId,
-      parentThreadId: outOfBandInvitation.id,
+      parentThreadId: outOfBandRecord.outOfBandInvitation.id,
```

Comment:
Ah we make it the `id` so the parentThreadId is always correct. Smart

Reply:
I thought this approach would be more appropriate because we would not be 'inventing` an OOB Invitation, but rather using a template based on the public DID. If someone looks at the OOBRecord, they will know that it comes from an implicit invitation because its OOB Invitation will have a did as id.